README.md

```diff
@@ -61,6 +61,10 @@ The default value is 1m.
 
     configsync.flush_interval: 1m
 
+### Reloading
+
+This plugin reloads the user dictionary automatically when the sync occurs.
```

Comment:
This plugin is not only for user dictionary files.
The master data for files is in an index.
So, a sync occurs when it's updated.